packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -21,12 +22,23 @@ import 'overlay.dart';
 /// mixed text: '<the nwor<b quick fox'
 ///   Here 'the b' is selected, but 'brown' is RTL. Both are drawn with the
 ///   [left] type.
```

Comment:
this comment renders... not as intended.
http://docs.flutter.io/flutter/material/TextSelectionHandleType-class.html